.github/workflows/pull_request.yml

```diff
@@ -65,7 +65,7 @@ jobs:
           else
               ansible_package=ansible-core
           fi
-          python3 -m pip install $ansible_package==$ansible_version.* docker ansible-modules-hashivault
+          python3 -m pip install $ansible_package==$ansible_version.* docker git+https://github.com/stackhpc/ansible-modules-hashivault@stackhpc
```

Comment:
Note to self, just pulls this in: https://github.com/stackhpc/ansible-modules-hashivault/commit/ee6509c831695ab8d851f127ea895baeb9f36fab